_posts/2023-07-24-level-up-using-security-devices.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Level up your RubyGems Account Security with Security Devices
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+
+In a [previous blog post](https://blog.rubygems.org/2023/04/11/security-device-cli-support.html), we introduced the latest updates to our security features, including CLI support for security devices and future plans for the feature.
+
+We are excited to announce that WebAuthn is fully integrated and is recognized to be the preferred choice for multi-factor authentication (MFA). When users register a security device, they will now be provided with recovery codes and be able to configure their [multi-factor authentication level](https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels) without needing to enable time-based one time password (OTP) MFA.
+
+### Why use a Security Device?
+Security devices add an extra layer of protection beyond passwords. It utilizes the [WebAuthn standard](https://www.yubico.com/authentication-standards/webauthn/), which employs public-key cryptography to mitigate the risk of password-related attacks such as phishing or credential stuffing.
+
+They provide better security and convenience compared to their time-based one time password counterparts, making them a preferred choice for authentication.
```

Comment:
```suggestion
These devices not only provide enhanced security but also offer greater convenience compared to traditional time-based one-time password methods, making them a preferred choice for authentication.
```

Reply:
I'm not too sure about using `traditional` to describe OTP, so I omitted that but I took the rest of the suggested wording 👍